src/components/scene/embedded.js

```diff
@@ -4,18 +4,16 @@ var registerComponent = require('../../core/component').registerComponent;
  * Component to embed an a-frame scene within the layout of a 2D page.
  */
 module.exports.Component = registerComponent('embedded', {
-  dependencies: ['vr-mode-ui'],
-
   schema: {default: true},
 
   update: function () {
     var sceneEl = this.el;
     var enterVREl = sceneEl.querySelector('.a-enter-vr');
     if (this.data === true) {
-      enterVREl.classList.add('embedded');
+      if (enterVREl) { enterVREl.classList.add('embedded'); }
```

Comment:
The problem si that it's not guaranteed that the enterVREl element it's going to be ready when the `update` method runs, hence the `vr-mode-ui` dependency